experimental/Schemes/ProjectiveSchemes.jl

```diff
@@ -186,11 +186,11 @@ On ``X ⊂ ℙʳ(A)`` this returns ``r``.
 fiber_dimension(P::ProjectiveScheme) = P.r
 
 @Markdown.doc """
-    homogeneous_coordinate_ring(X::ProjectiveScheme)
+    homog_poly_ring(X::ProjectiveScheme)
```

Comment:
Just so that I don't forget it, why the change from `homogeneous` to `homog`? Or is internal (then it should not be exported).